modules/tactical/src/idb.ts

```diff
@@ -0,0 +1,58 @@
+/// <reference path="../../../typings/rx/rx.d.ts" />
+/// <reference path="../../../typings/rx/rx-lite.d.ts" />
+
+
+import {Observable} from 'rx';
+
+/**
+ * A very minimal interface for a key-value store, designed as an abstraction
+ * layer over IndexedDB.
+ */
+export interface Idb {
```

Comment:
If in the future we would want to allow db adapters, should this interface be named something less specific to indexDB?